core/src/test/java/org/elasticsearch/script/expression/ExpressionScriptTests.java

```diff
@@ -385,8 +388,63 @@ public void testStringSpecialValueVariable() throws Exception {
             message = e.toString();
         }
         assertThat(message + "should have contained ExpressionScriptExecutionException",
-                   message.contains("ExpressionScriptExecutionException"), equalTo(true));
+                message.contains("ExpressionScriptExecutionException"), equalTo(true));
         assertThat(message + "should have contained text variable error",
-                   message.contains("text variable"), equalTo(true));
+                message.contains("text variable"), equalTo(true));
+    }
+
+    public void testExecutableScripts() throws Exception {
```

Comment:
While I would rather see these tests in a unit test (single node test), I understand that is a little difficult here. Can you at least add a test which actually runs a scripted aggregation?